CHANGELOG.md

```diff
@@ -25,11 +25,11 @@ Jump in to this release, with the new demos and examples:
 ### Major features and improvements
 
 - Better demonstration notebooks and documentation to make the library more accessible to new and existing users:
-  - Notebooks are [now published in the API documentation][demos-1.0.0], for better & faster rendering and more convenient access [\#1279](https://github.com/stellargraph/stellargraph/pull/1279) [\#1433](https://github.com/stellargraph/stellargraph/pull/1433) [\#1448](https://github.com/stellargraph/stellargraph/pull/1448)
+  - Notebooks are [now published in the API documentation][demos-1.0.0], for better & faster rendering and more convenient access [\#1279](https://github.com/stellargraph/stellargraph/pull/1279) [\#1433](https://github.com/stellargraph/stellargraph/pull/1433) [\#1448](https://github.com/stellargraph/stellargraph/pull/1448). Note that some notebooks in the repo are currently omitted from the API docs until they're reviewed further [\#1477](https://github.com/stellargraph/stellargraph/pull/1477)
```

Comment:
I think it's probably ok to not mention the non-visible notebooks here? It seems a bit like the wrong foot start with.

```suggestion
  - Notebooks are [now published in the API documentation][demos-1.0.0], for better & faster rendering and more convenient access [\#1279](https://github.com/stellargraph/stellargraph/pull/1279) [\#1433](https://github.com/stellargraph/stellargraph/pull/1433) [\#1448](https://github.com/stellargraph/stellargraph/pull/1448).
```

Or maybe:

```suggestion
  - Most notebooks are [now published in the API documentation][demos-1.0.0], for better & faster rendering and more convenient access [\#1279](https://github.com/stellargraph/stellargraph/pull/1279) [\#1433](https://github.com/stellargraph/stellargraph/pull/1433) [\#1448](https://github.com/stellargraph/stellargraph/pull/1448).
```

Reply:
Yeah good point, I'll just remove it